tinylog-impl/src/main/java/org/tinylog/writers/RollingFileWriter.java

```diff
@@ -96,7 +100,39 @@ public RollingFileWriter(final Map<String, String> properties) throws FileNotFou
 		charset = getCharset(properties);
 		buffered = Boolean.parseBoolean(properties.get("buffered"));
 		writingThread = Boolean.parseBoolean(properties.get("writingthread"));
-		writer = createByteArrayWriter(fileName, append, buffered, false, false);
+		writer = createByteArrayWriterAndLinkLatest(fileName, append, buffered, false, false);
+	}
+
+	private static List<File> filterOutSymlinks(final List<File> files) {
+		if (!RuntimeProvider.isAndroid()) {
+			List<File> symlinks = new ArrayList<File>();
+			for (File file : files) {
+				if (java.nio.file.Files.isSymbolicLink(file.toPath())) {
+					symlinks.add(file);
+				}
+			}
+			files.removeAll(symlinks);
+		}
+		return files;
+	}
+
+	private ByteArrayWriter createByteArrayWriterAndLinkLatest(final String fileName, final boolean append, final boolean buffered,
+		final boolean threadSafe, final boolean shared) throws FileNotFoundException {
+		ByteArrayWriter writer = AbstractFormatPatternWriter.createByteArrayWriter(fileName, append, buffered, threadSafe, shared);
```

Comment:
The class name `AbstractFormatPatternWriter` is redundant here, you can remove it to make the line a bit shorter